plugins/github-actions/src/components/Router.tsx

```diff
@@ -41,7 +41,7 @@ export const Router = () => {
   }
   return (
     <Routes>
-      <Route path="/" element={<WorkflowRunsTable entity={entity} />} />
+      <Route path="/" element={<WorkflowRunsCard entity={entity} />} />
```

Comment:
From my reading this change would mean you would no longer be able to use the Table view. I'm not sure that everyone using this plugin would want that. Can you update this so both views are supported? That gives people the option to pick the one they want

Reply:
> From my reading this change would mean you would no longer be able to use the Table view. I'm not sure that everyone using this plugin would want that. Can you update this so both views are supported? That gives people the option to pick the one they want

Like mentioned above its only a option either table view or card view can be used here.., its our choice to use either of those, may be we can detailed the usage in ReadMe file, can you suggest??